src/assets/style/module/airport/_airport.less

```diff
@@ -1,11 +1,11 @@
 /* TODO: this rule should be moved to a class instead of ID */
 /* TODO: this rule should be broken up into several other rules */
 #airport-switch {
+    width: fit-content;
```

Comment:
this is unnecessary and unsupported (https://caniuse.com/#search=fit-content)

Reply:
Without it, it doesn't achieve the desired effect, so I'll wait until we can find another way to do it. I stink at CSS, and am surprised I could even get this to work at all :wink:

The idea is for the width to be only as wide as the content inside needs it to be, and allows it to expand its width up to 600px.

Reply:
yeah, it does that already even without the `fit-content`.  what, exactly, is the desired effect?

also, how big is your screen?

Reply:
The desired effect is for `#airport-switch` to be no wider than its content necessitates, and for it to expand as needed to fit longer airport names, with a maximum width of 600px. Currently, it expands and tops out at 600px, but does not shrink beyond a certain width, even if the airport names are very short. (Note that the bottom note text can be ignored, as it will be removed soon)

![image](https://user-images.githubusercontent.com/5103735/42910871-f8566430-8ab6-11e8-8728-8371b912cf82.png)

Reply:
And I was originally on my big monitor, 2560x1440, now on the laptop (where the screenshot was taken), 1920x1080.

Reply:
@n8rzz Open to other ways to resolve this, or even just rolling that into another issue, #1025 (basically the same problem in the other axis), and having this PR discard these CSS changes, and simply increase the max-width.